.github/workflows/release.yml

```diff
@@ -104,7 +134,7 @@ jobs:
           #update sdk to latest snapshot
           mvn versions:update-properties  -Dincludes=org.ehrbase.openehr.sdk:*  -DallowSnapshots=true
           #edit changelog
-          sed -i '5i ## [unreleased]\n ### Added \n ### Fixed \n' CHANGELOG.md
+          sed -i '7i ## [unreleased]\n ### Added\n ### Changed \n ### Fixed \n' CHANGELOG.md
```

Comment:
Should it be "8i" ?

Reply:
no. I tested it 
https://github.com/stefanspiska/ehrbase/blob/develop/CHANGELOG.md?plain=1

Reply:
To me it looks like line 7 should be a blank line instead of line 17

Reply:
The double blank line is not produced by this script.

Reply:
Also @HolgerReiseVSys you are rigth it should be 8i if we assume line 7 to be blank. which actually produced the one time double blank line